run.go

```diff
@@ -734,3 +746,18 @@ func propagateContainerdLabelsToOCIAnnotations() oci.SpecOpts {
 		return oci.WithAnnotations(c.Labels)(ctx, oc, c, s)
 	}
 }
+
+func writeCIDFile(path, id string) error {
+	if _, err := os.Stat(path); err == nil {
+		return errors.Errorf("Container ID file found, make sure the other container isn't running or delete %s", path)
+	}
+	f, err := os.Create(path)
+	defer f.Close()
+	if err != nil {
+		return errors.Errorf("Failed to create the container ID file: %s", err)
```

Comment:
nit: error string should start with a lower character in Go

Reply:
The error message was inspired from https://github.com/docker/cli/blob/master/cli/command/container/create.go#L180, changed it to lowercase though